DokanNet/Native/DOKAN_OPTIONS.cs

```diff
@@ -8,7 +8,7 @@ namespace DokanNet.Native
     /// <see cref="NativeMethods.DokanMain"/>
     /// <remarks>This is the same structure as <c>PDOKAN_OPTIONS</c> (dokan.h) in the C version of Dokan.</remarks>
     [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Unicode, Pack = 4)]
-    internal sealed class DOKAN_OPTIONS
+    public sealed class DOKAN_OPTIONS
```

Comment:
This now public due to OptionsConfigurationDelegate ? I guess that ok. Looking at all the fields and you change it is now all "c#" members 👍

Reply:
Yep, could hide it with a little more work, but I didn't see any advantage